### DISCUSSION TITLE
.NET MAUI vs WinUI 3

### DISCUSSION THREAD

**Discussion by yairp03** [dotnet/maui#8409] [Q&A]
I want to develop a Windows desktop app, should I use MAUI or WinUI 3?
On the one hand, I'm already familiar with MAUI structure, syntax, and controls, because I built some apps with it.
On the other hand, maybe MAUI is overkill / too much / less performant than WinUI 3 if I intend to only target Windows.
Also, I'm guessing I'll lose some of WinUI 3 features if I'll use MAUI because it brings only the shared features between the platforms.

**Comment by francoistanguay** [comment]
FYI https://github.com/dotnet/maui/issues/6791

Also, if you think you'll want to leverage anything from the WinUI Community Toolkit, or any WinUI control not available through MAUI directly, make sure you understand how can this be done, if at all.

I haven't found any docs/samples on how to do so and am unclear if it's even supported.

**Reply by jindal1979** [reply]
I have done lot of experiments. Its better to do Windows development in WinUI 3 rather than in MAUI. The MAUI will restrict you a lot or will make it difficult to use WinUI 3 controls or community libraries. 

I had the same question in my mind last month and I decided to use WinUI 3.

**Reply by GeraudFabien** [reply]
If you want you can also check [Uno platform](https://platform.uno/). It bring WinUI on all platform including Web and Linux. Platform Maui has not support.

**Reply by yairp03** [reply]
Yeah I've heard about it, but my interest is about when I target only Windows

**Comment by coderbasket** [comment]
Check out more sample here for any .Net Projects including Avalonia.
[https://coderbasket.github.io/framework/Avalonia](https://coderbasket.github.io/framework/Avalonia)
